src/preload-hover.js

```diff
@@ -12,20 +12,23 @@ export default class PreloadHover {
   }
 
   start(domScope = this.configuration.defaultDomScope) {
-    if (!domScope) { throw new Error('domScope must be provided.'); }
+    if (!domScope) { throw new Error('domScope must be provided.'); } 
+    domScope = [...new Set(domScope)];
```

Comment:
De-duplicating the provided domScope items may not be necessary. However, this pattern should be probably to the `links` -- as in combine all the links together in one array and de-duplicate them from there.